tests/jax/test_custom_call_compute.py

```diff
@@ -19,8 +19,10 @@
 from transformer_engine.jax.fp8 import FP8MetaPackage, FP8Helper, is_fp8_available
 from transformer_engine.jax.layernorm import layernorm, layernorm_fp8_dot
 from transformer_engine.jax.layernorm_mlp import activation_lu, fused_layernorm_fp8_mlp
+from transformer_engine.jax.cpp_extensions.activation import _convert_to_activation_function
```

Comment:
Hi, how about importing and using `_jax_act_lu` instead of `_convert_to_activation_function`?

Reply:
done